jobs/config.json

```diff
@@ -1685,23 +1698,29 @@
   "scenario": "kubernetes_e2e",
   "args": [
     "--env-file=platforms/gke.env",
-    "--env-file=jobs/ci-kubernetes-e2e-gke-large-cluster.env"
+    "--env-file=jobs/ci-kubernetes-e2e-gke-large-cluster.env",
+    "--cluster=gke-large-cluster"
   ]
 },
 
 "ci-kubernetes-e2e-gke-large-deploy": {
   "scenario": "kubernetes_e2e",
   "args": [
     "--env-file=platforms/gke.env",
-    "--env-file=jobs/ci-kubernetes-e2e-gke-large-deploy.env"
+    "--env-file=jobs/ci-kubernetes-e2e-gke-large-deploy.env",
+    "--cluster=gke-large-deploy",
+    "--down=false"
   ]
 },
 
 "ci-kubernetes-e2e-gke-large-teardown": {
   "scenario": "kubernetes_e2e",
   "args": [
     "--env-file=platforms/gke.env",
-    "--env-file=jobs/ci-kubernetes-e2e-gke-large-teardown.env"
+    "--env-file=jobs/ci-kubernetes-e2e-gke-large-teardown.env",
+    "--cluster=gke-large-deploy",
+    "--up=false",
+    "--test=false"
```

Comment:
From scalability-related jobs it LGTM.
However, you modified 16 ".env" files, but only 13 jobs here, so something is probably missing...